Automattic/jetpack
# Spinner component: Replace with Core's spinner

Replaces the broken spinner component we've been using. 

#### Changes proposed in this Pull Request:

Replaces our custom `<Spinner />` component with [the one from Core](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/spinner) 

I removed the spinner entirely from the Search bar in /settings, because it didn't seem to be rendering anyway. 
After this change, we're only using the spinner in three places. 
- Stats area in /dashboard
- Creativemail install button in the /recommendations
- Termination (disconnect) dialoge

**Stats :: Before:**
![image](https://user-images.githubusercontent.com/7129409/113646493-eba8e380-9656-11eb-80e8-38b8aa424b76.png)

**Stats:: After**
![image](https://user-images.githubusercontent.com/7129409/113790761-1dca4c00-9710-11eb-88ad-9fddd8a95615.png)

#### Jetpack product discussion


p1617650857426000-slack-jetpack-crew

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






Stats area:
- When the At A Glance dashboard is loading, you should see the spinner in the stats area. 

Install Button: 
- Looking at the code, I think this should be displaying somewhere in the /recommendations when someone clicks to install Creativemail. I'm not really sure how to get there to test it TBH, but it seems like the CSS removed from projects/plugins/jetpack/_inc/client/components/install-button/style.scss won't impact the positioning of the spinner at all, and I didn't see any other custom styling in there for `dops-spinner`. 

Termination dialogue 
- AKA the disconnect modal. We show a spinner there before the dialog pops up. You'll see it for a brief moment when you click "manage site connection" for the first time on a page load. Too quick to even screenshot it.
